packages/next-dashboard/src/components/Statistic.js

```diff
@@ -31,14 +32,29 @@ function Statistic({
   direction,
   className,
   isLoading = false,
+  positive = 'up',
   ...rest
 }: Props): R.Node {
+  let isNegative = false;
+  let isPositive = false;
+  if (positive === 'down') {
+    isPositive = direction === 'down';
+    isNegative = direction === 'up';
+  } else if (positive === 'up') {
+    isPositive = direction === 'up';
+    isNegative = direction === 'down';
+  }
```

Comment:
```suggestion
  let isNegative = false;
  let isPositive = false;
  if (positive != null) {
    isPositive = direction === positive;
    isNegative = direction === positive;
  }
```

Reply:
Ah, right! Or just `const isPositive = positive != null && direction === positive;` and conversely for `isNegative`. But I'm happy with `let`s and an `if` too!